test/integration/drag_and_drop_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+class DragAndDropTest < ActionDispatch::IntegrationTest
+  before do
+    Capybara.current_driver = :poltergeist
+    @user = create :user
+    log_in_as @user
+
+    4.times do |i|
```

Comment:
Took a look at this and I was having it choke on the `before` block -- `table_content` uses some stuff on the pregnancy object (specifically LMP), and pregnancy objects didn't seem to be being created. 

This is probably a signal that we should either a) adjust the factory to create pregnancy when we create a patient or b) write a validation such that making a patient without a pregnancy chokes. Or maybe neither! Thoughts?